iroha-cli/main.cpp

```diff
@@ -158,20 +156,17 @@ int main(int argc, char *argv[]) {
       return EXIT_FAILURE;
     }
     iroha::KeysManagerImpl manager((path / FLAGS_account_name).string());
-    boost::optional<iroha::keypair_t> keypair;
-    if (FLAGS_pass_phrase.size() != 0) {
-      keypair = manager.loadKeys(FLAGS_pass_phrase);
-    } else {
-      keypair = manager.loadKeys();
-    }
+    auto keypair = FLAGS_pass_phrase.size() != 0
+        ? manager.loadKeys(FLAGS_pass_phrase)
+        : manager.loadKeys();
     if (not keypair) {
       logger->error(
           "Cannot load specified keypair, or keypair is invalid. Path: {}, "
-          "keypair name: {}. Use --key_path to path to your keypair. \nMaybe wrong pass phrase (\"{}\")?",
+          "keypair name: {}. Use --key_path to path to your keypair. \n"
```

Comment:
fix the typo as well plz: `--key_path to path to your keypair` -> `--key_path with path of your keypair`